package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-ui-kit",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-ui-kit/issues",
-  "version": "5.3.0",
+  "version": "6.3.1",
```

Comment:
I'm ok with updating peer deps -> breaking release but then we need to do the following:
- start at 0: `6.0.0`
- update `CHANGELOG.md`